src/planner/macos/profiles.rs

```diff
@@ -0,0 +1,158 @@
+use std::collections::HashMap;
+
+use crate::execute_command;
+
+#[derive(thiserror::Error, Debug)]
+pub enum LoadError {
+    #[error("Profile plist parsing error: {0}")]
+    Parse(#[from] plist::Error),
+
+    #[error("Profile discovery error: {0}")]
+    ProfileListing(#[from] crate::ActionErrorKind),
+}
+
+pub async fn load() -> Result<Policies, LoadError> {
+    let buf = execute_command(
+        tokio::process::Command::new("/usr/bin/profiles")
+            .args(["-P", "-o", "stdout-xml"])
```

Comment:
Does this binary have long-form flags? If so, I would prefer using those over the short flags.

Reply:
We can only dream.

Reply:
wtf macos :sob:

Reply:
Could I be greedy and ask that you add a comment after the flag with what it does (from the `--help` output)?

Like:

```suggestion
            // "prints all configuration profiles to console"
            .arg("-P")
            // "path to output XML plist file (for -P, -L, -C).  Use 'stdout' to send information to the console."
            // NOTE(grahamc): `stdout` doesn't output XML formatting, but `stdout-xml` does
            .args(["-o", "stdout-xml"])
```